cmd/issuer-rest/startcmd/start.go

```diff
@@ -826,3 +846,19 @@ func getOauth2Config(authURL, tokenURL, redirectURL, clientID, secret string) *o
 
 	return config
 }
+
+func readProfiles(path string) ([]*operation.Profile, error) {
+	f, err := os.Open(path) //nolint:gosec
+	if err != nil {
+		return nil, fmt.Errorf("open file with profiles: %w", err)
```

Comment:
If an error happens here, the full error message that gets wrapped up will be `read profiles: open file with profiles: <some underlying err message here>`

Notice the repetition of "profiles" in the error message? You can reduce that by simplifying the error message here on line 853 to be specific to the scope of the operation. Consider changing the error to `"open file: %w"`.

Now your wrapped error will be ``read profiles: open file: <some underlying err message here>`, which is cleaner, less redundant, and easier to read.

Reply:
Makes sense, thanks for the insight - updated!